### PR TITLE
Fix assert <built-in function callable>

### PR DESCRIPTION
**Pull request by jaryn**
addFinalizer needs callable and not the result of the call.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a93581746ca8a2395138335d2b2a462a4090b13c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/containers/test_provider_crud.py [hunk 1]
```diff
@@ -42,5 +42,5 @@ def test_provider_edit_port(request, a_container_provider):
     old_port = a_container_provider.port
     with update(a_container_provider):
         a_container_provider.port = '1234'
-    request.addfinalizer(a_container_provider.update({'port': old_port}))
+        request.addfinalizer(lambda: a_container_provider.update({'port': old_port}))
```
Comment (psav) [inline cfme/tests/containers/test_provider_crud.py]: I think that this request should go straight after ```old_port```, but I agree that the result of the callable is not the right way to go, but as we are fixing it, let's move it to where it should be.

**Comment by dajoRH** [comment]
Lint report for commit 1a07637e1b3aed132ff43618b3f96ac813633b2b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 57a9d35b42787b1f77f7720e64c0a840e4a46485:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
Looked at the error, none related to the fix.

**Pull request merged**